src/data/queries/tests/pressRelease.test.tsx

```diff
@@ -43,14 +43,48 @@ describe(`${RESOURCE_TYPES.PRESS_RELEASE} formatData`, () => {
     expect(formattedData.administration.name).toBeNull()
     expect(formattedData.pdfVersion).toBeNull()
   })
+  test('handles missing or null contact fields correctly', () => {
+    const modifiedMockContact = {
```

Comment:
I recommend assigning this the expected type:
```suggestion
    const modifiedMockContact: NodePressRelease = {
```